conanfile.py

```diff
@@ -22,7 +22,8 @@ class OrbitConan(ConanFile):
                        "fPIC": True}
     _orbit_channel = "orbitdeps/stable"
     exports_sources = "CMakeLists.txt", "Orbit*", "bin/*", "cmake/*", "external/*", "LICENSE"
-    build_requires = ('grpc_codegen/1.27.3@orbitdeps/stable',
+    build_requires = ('breakpad/216cea7b@orbitdeps/stable',
+                      'grpc_codegen/1.27.3@orbitdeps/stable',
```

Comment:
We shouldn't include `breakpad` in Orbit's build process just to make it available. Most devs will never need it and are now required to download it.

The better way is to introduce a conan profile which requires breakpad as a build requirement. But at first I suggest we go with manual installing of breakpad via `conan install -g deploy breakpad/216cea7b@orbitdeps/stable` (This would install the breakpad binaries into the current directory.)

Reply:
Removed all breakpad dependencies from this file